google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudDebugProcess.java

```diff
@@ -261,9 +261,12 @@ public void run(@NotNull ProgressIndicator indicator) {
     // Start breakpoints refresh job on first use.
     getStateController().addListener(this);
     getStateController().startBackgroundListening();
-    SwingUtilities.invokeLater(new Runnable() {
-      @Override
-      public void run() {
+    SwingUtilities.invokeLater(() -> {
+      // If a breakpoint handler has not been created before, getBreakpointHandler() will attempt to
+      // create one. But if the project is disposed of, this creation will fail because it will try
+      // to access the project's components when creating the PsiManager. This guards against a
+      // failure in this scenario.
+      if (breakpointHandlers != null || !getXDebugSession().getProject().isDisposed()) {
```

Comment:
It likely is, but the `|| !getXDebugSession().getProject().isDisposed()` clause will ensure it will still run, as long as the project hasn't been disposed. I can't imagine anything is depending on this executing even when the project is disposed because they would be seeing the `AssertionError` thrown just like this test.